include/aos/common/cryptoutils.hpp

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_CRYPTOUTILS_HPP_
+#define AOS_CRYPTOUTILS_HPP_
+
+#include "aos/common/crypto.hpp"
+#include "aos/common/pkcs11/pkcs11.hpp"
+#include "aos/common/uuid.hpp"
+
+namespace aos {
+namespace cryptoutils {
+
+class Helper {
+public:
+    /**
+     * Constructs object instance.
+     *
+     * @param cryptoProvider crypto provider interface.
+     * @param pkcs11Context PKCS11 session context.
+     * @param uuidManager uuid manager.
+     * @param allocator allocator.
+     */
+    Helper(crypto::x509::ProviderItf& cryptoProvider, pkcs11::SessionContext& pkcs11Context,
+        uuid::UUIDManagerItf& uuidManager, Allocator& allocator);
+
+    /**
+     * Loads certificate chain by URL.
+     *
+     * @param url input url.
+     * @return RetWithError<SharedPtr<crypto::x509::CertificateChain>>.
+     */
+    RetWithError<SharedPtr<crypto::x509::CertificateChain>> LoadCertsChainByURL(const String& url);
+
+    /**
+     * Loads private key by URL.
+     *
+     * @param url input url.
+     * @return RetWithError<SharedPtr<crypto::PrivateKeyItf>>.
+     */
+    RetWithError<SharedPtr<crypto::PrivateKeyItf>> LoadPrivKeyByURL(const String& url);
+
+private:
+    using PEMCertChainBlob = StaticArray<uint8_t, crypto::cPEMCertSize * crypto::cCertChainSize>;
+
+    RetWithError<SharedPtr<crypto::x509::CertificateChain>> LoadCertsFromFile(const String& fileName);
+    RetWithError<SharedPtr<crypto::PrivateKeyItf>>          LoadPrivKeyFromFile(const String& fileName);
+
+    crypto::x509::ProviderItf& mCryptoProvider;
+    pkcs11::SessionContext&    mPKCS11Context;
+    uuid::UUIDManagerItf&      mUUIDManager;
+
+    Allocator& mAllocator;
+};
+
+/**
+ * File scheme.
+ */
+constexpr auto cSchemeFile = "file";
```

Comment:
Move const to the top of file. Also looks like these consts are private for `Helper` class.

Reply:
constants moved to cpp file